index.js

```diff
@@ -344,6 +344,9 @@ class WebdriverAjax {
     // (best effort compliance with RFC)
     function parseResponseHeaders(rawHeader) {
       const headers = {};
+      if (rawHeader === '') {
+        return headers;
```

Comment:
Can we get some logging for this scenario, e.g. `"${transformed.method} request to ${req.url} (HTTP ${req.statusCode}) had no response headers!"`, [here](https://github.com/webdriverio-community/wdio-intercept-service/blob/0162292cb2bb7c0761cefb0f37881e233ff0fb17/index.js#L330-L331)? That way people can at least see how often this happens. We could suggest reporting the request here, too.